internal/test/test.go

```diff
@@ -47,3 +49,16 @@ func CleanupConfig() {
 	config.SetOrgID("")
 	config.SetProjectID("")
 }
+
+// VerifyTemplate validates that the given template string is valid.
+func VerifyOutputTemplate(t *testing.T, templateString string, expected interface{}) {
```

Comment:
```suggestion
func VerifyOutputTemplate(t *testing.T, template string, expected interface{}) {
```
no need to suggest the type, that's what the compiler is for

Reply:
Reason: golangcilint reports mixed import and variable name.

templateInput is ok?

Reply:
Naming is hard :)

Reply:
Very clear! Applying it.
The reason we have this helper is to not be strongly dependent on the library we use.

Reply:
yeah, this helper makes perfect sense 💯